src/main/java/org/spongepowered/api/service/pagination/PaginationList.java

```diff
@@ -103,8 +103,18 @@ static Builder builder() {
      * Send the constructed pagination list to the given source.
      *
      * @param source The source to send to
+     * @param page The page to send
```

Comment:
We should mention that the first page is one vs zero as developers count indexes. We should also mention what happens if we use a page after the last one.